### PR TITLE
Desktop version project

### PR DESCRIPTION
**Pull request by haroldmud**
In this PR I did the following :point_down: 

- Create the website using HTML
- Add the style on it using CSS
- Extract the SVG files for more decoration
- Add some animations and transition from pair programming

### CODE REVIEW

File: milestone.css [hunk 1]
```diff
@@ -569,23 +564,47 @@ a:hover {
...
+
+.name_input1 {
+  width: 220px;
+}
+
+@media (max-width) {
+  .name_input1 {
+    width: 307px;
+  }
+}
```
Comment (bimbolabuari) [inline milestone.css:579]: You need to give a defined width for the media query for this to work fine  👍 

Example:
```
@media screen and (min-width: 990px) 
```

File: milestone.css [hunk 2]
```diff
@@ -623,6 +642,25 @@ textarea {
   margin-top: 40px;
 }
 
+.button-sub {
+  display: none;
+}
+
+@media (max-width: 950px) {
```
Comment (bimbolabuari) [inline milestone.css:649]: For CSS best practice, You need to define the media query for a specific breakpoint once.

Kindly remove all repetitions 👍 

https://github.com/haroldmud/Microverse-milestone-project/blob/388d5e9e33184ec531ecbce71acec092f4f29a5e/milestone.css#L1

https://github.com/haroldmud/Microverse-milestone-project/blob/388d5e9e33184ec531ecbce71acec092f4f29a5e/milestone.css#L23

https://github.com/haroldmud/Microverse-milestone-project/blob/388d5e9e33184ec531ecbce71acec092f4f29a5e/milestone.css#L33

https://github.com/haroldmud/Microverse-milestone-project/blob/388d5e9e33184ec531ecbce71acec092f4f29a5e/milestone.css#L71

https://github.com/haroldmud/Microverse-milestone-project/blob/388d5e9e33184ec531ecbce71acec092f4f29a5e/milestone.css#L107

Kindly check for other occurrences 🙂

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Use both Flexbox and Grid in order to layout the elements on the page

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Deepakdanger — APPROVED** [review]
Hi @haroldmud, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**